examples/traverse-ipld-graphs/eth.js

```diff
@@ -20,34 +18,25 @@ createNode((err, ipfs) => {
     path.join(__dirname, '/eth-blocks/block_302517')
   ]
 
-  asyncEach(ethBlocks, (ethBlockPath, cb) => {
-    const data = fs.readFileSync(ethBlockPath)
-
-    multihashing(data, 'keccak-256', (err, multihash) => {
-      if (err) {
-        cb(err)
-      }
-      const cid = new CID(1, 'eth-block', multihash)
-      // console.log(cid.toBaseEncodedString())
-
-      ipfs.block.put(new Block(data, cid), cb)
-    })
-  }, (err) => {
-    if (err) {
-      throw err
-    }
-
-    const block302516 = 'z43AaGEywSDX5PUJcrn5GfZmb6FjisJyR7uahhWPk456f7k7LDA'
-    const block302517 = 'z43AaGF42R2DXsU65bNnHRCypLPr9sg6D7CUws5raiqATVaB1jj'
-
-    function errOrLog (err, result) {
-      if (err) {
-        throw err
-      }
-      console.log(result.value.toString('hex'))
-    }
-
-    ipfs.dag.get(block302516 + '/number', errOrLog)
-    ipfs.dag.get(block302517 + '/parent/number', errOrLog)
-  })
-})
+  for (let i = 0; i < ethBlocks.length; i++) {
+    const data = await fs.readFile(ethBlocks[i])
+    const multihash = await promisify(multihashing)(data, 'keccak-256')
```

Comment:
If you use the most recent version of `multihashing-async` you wouldn't need to promisify it.

Reply:
There's no `package.json` in this example so we have to use the version that's installed in the repo.